modules/SofaGraphComponent/SceneCheckerVisitor.cpp

```diff
@@ -62,7 +62,7 @@ void SceneCheckerVisitor::addHookInChangeSet(const std::string& version, ChangeS
 void SceneCheckerVisitor::installChangeSets()
 {
     addHookInChangeSet("17.06", [](Base* o){
-        if(o->getClassName() == "RestShapeSpringsForceField" && o->findData("external_rest_shape")->isSet())
+        if(o->getClassName() == "RestShapeSpringsForceField" && o->findLink("external_rest_shape")->getSize() != 0)
             msg_warning(o) << "RestShapeSpringsForceField have changed since 17.06. The parameter 'external_rest_shape' is now a Link. To fix your scene you need to add and '@' in front of the provided path. See PR#315" ;
```

Comment:
What we want to check is if the object still have "something" with the name "external_rest_shape" and if this thing has be set by the user. Before the merge of PR 315 this thing was a Data, but since the PR it is a Link. 

So I think searching for the link is the right thing to do.